### ISSUE TITLE
E117: Unknown function: mundo#MundoShow

### ISSUE THREAD

**Issue by lopippo** [simnalamburt/vim-mundo#56]
Greetings,
I was happily using Gundo when suddenly it failed working. I have researched the web and found vim-mundo with new energies and coders. So I made the switch. But there still seems to be a problem: when I enter :MundoToggle I get the following error message:
E117: Unknown function: mundo#MundoToggle.

Same for MundoShow (see subject line).
I am on Debian, my vim is effectively built with support for python:
vim --version gives -L/usr/lib/python3.5/config-3.5m-x86_64-linux-gnu
and I have python2.7 and python3 installed.

What can I do to fix this ?
Thank you for your endeavours,
Filippo

**Comment by dsummersl** [comment]
Could you run a couple things and let us know what you get, in vim?

`:echo has('python')` (sounds like...you would see 0).
`:echo has('python3')` (would expect this to return 1).

Try adding this to your vimrc file:

`let g:mundo_prefer_python3 = 1`

**Comment by lopippo** [comment]
Greetings, 
thank you for your quick reply.
:echo has('python') gives indeed 0
:echo has('python3') gives indeed 1

But adding the line you suggest did not solve the problem.

I have installed Mundo-Vim using Vundle, by adding this line to my .vimrc:

Plugin 'simnalamburt/vim-mundo'

then I ran :Plugin-Install, which could install the plugin without any error.
Then, I can see in my .vim the following:

$ ls $HOME/.vim/bundle/vim-mundo/
autoload/         doc/              .git/             .gitignore        plugin/           README.md         requirements.txt  tests/

Does all this look correct ?

Thank you,
Filippo

**Comment by dsummersl** [comment]
It sounds correct...

Try...starting vim, launching mundo (`:MundoToggle`), and then printing out the python system path (`:py3 print(sys.path)`)

It should look something like:

```
[..., '~/vim/bundle/gundo.vim/autoload/mundo', '~/vim/bundle/vim-mundo/autoload', ...] 
```

**Comment by lopippo** [comment]
Greetings, 
I think I got it solved! What was messing around, I guess, is that I use to lauch vim with -S Session.vim automatically. In my work session I indeed had a Session.vim file.

So, when trying your suggestion, I actually started a clear vim (no session file load) and :Mundo-Toggle actually opened the side view as I usually expected Gundo to do it! Anyways, I had removed the gundo bundle directory before, so it does not show up with :py3 print(sys.path). This is what I get: 
'.vim/bundle/vim-mundo/autoload/mundo', .vim/bundle/vim-mundo/autoload'

Your can call this issue done, thank you soo muuch for your responsiveness.
Cheers,
Filippo
